lib/widgets/navigation_bar/interactive_nav_item.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:universal_html/html.dart' as html;
+
+class InteractiveNavItem extends MouseRegion {
+  static final appContainer =
```

Comment:
is there maybe a better way to change the mouse pointer on hover? if not, then this is okay

Reply:
I simplified the mouse pointer on hover by using the built-in `cursor` property of `MouseRegion`, which made the `interactive_nav_item.dart` file redundant and was removed.